.jshintrc

```diff
@@ -49,7 +49,7 @@
     "lastsemic"     : false,       // Suppresses warnings about missing semicolons, but only when the semicolon is omitted for the last statement in a one-line block.
     "laxbreak"      : false,       // Suppresses most of the warnings about possibly unsafe line breaks in your code.
     "laxcomma"      : false,       // Suppresses warnings about comma-first coding style.
-    "loopfunc"      : false,       // Suppresses warnings about functions inside of loops.
+    "loopfunc"      : true,        // Suppresses warnings about functions inside of loops.
```

Comment:
It looks like [functions inside of loops can lead to some pretty nasty bugs](http://jshint.com/docs/options/#loopfunc). Is it worth it to flip this to `false` rather than correcting each occurrence?